ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/RefragmentationService.java

```diff
@@ -0,0 +1,7 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.entities.LdesFragment;
+
+public interface RefragmentationService {
```

Comment:
What is 'refragmentation'? Is this for creating fragments for a new view? Is this to refragment existing views to get rid of empty/semi-empty views? Multiple interpretations are possible with this name. Perhaps we should add some javadoc to say what this does and does not do?

Reply:
Or a different name that has no ambiguity is possible as well.